OpenSim/Simulation/Model/Marker.cpp

```diff
@@ -49,6 +49,18 @@ Station()
 
 //_____________________________________________________________________________
 /**
+* Conveniece constructor.
```

Comment:
Conveniece -> Convenience.
Also, it's unnecessary to use doxygen-style formatting (`/** ... */`) for comments in .cpp files.

Reply:
@nickbianco, you might know this already but the thing that makes it a doxygen comment is the two asterisks rather than 1 (`/**` vs `/*`).

Reply:
I'll fix the typo. @chrisdembia I didn't know that, I'll remove them from the comments too while I'm at it.